test/core/extensions/coreIssuanceOrder.spec.ts

```diff
@@ -242,22 +269,188 @@ contract('CoreIssuanceOrder', accounts => {
 
       const formattedLogs = await getFormattedLogsFromTxHash(txHash);
       const expectedLogs = getExpectedFillLog(
-        setToken.address,
-        signerAccount,
-        subjectCaller,
-        makerToken.address,
-        relayerAccount,
-        relayerToken.address,
-        subjectQuantityToIssue,
-        ether(10),
-        ether(3),
-        issuanceOrderParams.orderHash,
+        setToken.address,              // setAddress
```

Comment:
Since we have these comments here now...do we wanna just add the same ones to the log test for the partial fill scenario?